strings/group_anagrams.py

```diff
@@ -0,0 +1,58 @@
+from typing import List
```

Comment:
We only support the latest version of CPython so you can replace all occurrences of `List` with `list`